docs/docs/club.md

```
@@ -1,54 +1,220 @@
+- Educational materials and resources to facilitate learning
 
-### 12. Stay Engaged with FOSS United:
+#### B. Active Clubs:
+- Financial support of up to ₹50,000 per year
+- Expert speakers for lectures and workshops
+- Educational resources and learning materials
+- Mentorship for projects and activities
+- Top-performing clubs may receive an additional ₹50,000 to host FOSS Hack 
+
+#### How do active clubs receive their annual ₹50,000 support?
+For each event organized by your FOSS Club, FOSS United will provide support of up to ₹5,000 for refreshments per event. Additionally, FOSS United will cover travel expenses for invited speakers and mentors. For detailed information, visit: https://forum.fossunited.org/t/foss-event-standards/2004#p-3900-h-4-refreshment-5
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)